.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '00:00'
+  open-pull-requests-limit: 10
+  reviewers:
+    - adamwathan
+  assignees:
+    - adamwathan
+  commit-message:
+    prefix: fix
+    prefix-development: chore
+    include: scope
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: daily
+    time: '00:00'
+  open-pull-requests-limit: 10
+  reviewers:
+    - adamwathan
+  assignees:
+    - adamwathan
+  commit-message:
+    prefix: fix
+    prefix-development: chore
+    include: scope
```

Comment:
```suggestion
version: 2
updates:
- package-ecosystem: npm
  directory: "/"
  schedule:
    interval: daily
    time: '00:00'
  open-pull-requests-limit: 10
  reviewers:
    - adamwathan
  assignees:
    - adamwathan
  commit-message:
    prefix: fix
    prefix-development: chore
    include: scope

- package-ecosystem: github-actions
  directory: "/"
  schedule:
    interval: daily
    time: '00:00'
  open-pull-requests-limit: 10
  reviewers:
    - adamwathan
  assignees:
    - adamwathan
  commit-message:
    prefix: fix
    prefix-development: chore
    include: scope
```

Reply:
I've added comments instead of adding spaces or new lines for better differentiation of the ecosystems